web-ui/src/main/resources/catalog/js/GnAdminMenu.js

```diff
@@ -0,0 +1,47 @@
+(function() {
+  goog.provide('gn_admin_menu');
+
+  var module = angular.module('gn_admin_menu',[]);
+
+  module.constant('$LOCALES', ['core', 'admin']);
```

Comment:
GnAdminModule already declare this constant. See https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/js/GnAdminModule.js#L22. Not sure this is usefull here ?